src/views/preview/project-view-hoc.jsx

```diff
@@ -0,0 +1,88 @@
+const bindAll = require('lodash.bindall');
+const React = require('react');
+const PropTypes = require('prop-types');
+const connect = require('react-redux').connect;
+
+const storage = require('../../lib/storage.js').default;
+const previewActions = require('../../redux/preview.js');
+
+const Sentry = require('@sentry/browser');
```

Comment:
@rschamp, please confirm that putting this code here makes sense!

Reply:
It is a little odd, since this file _could_ be required by multiple components and then the act of requiring the file would have side effects (although the way webpack works, I believe this code would only run the first time that happened...). It would be less surprising if this were in the `componentDidMount` of the HOC, or if you wanted it to run earlier than that, just before you define `class WrappedComponent`, so it would only run when the HOC was actually used.

Reply:
Moving it into wrapper like you suggested

Reply:
Moved this into lib